docs/GettingStarted.md

```diff
@@ -25,6 +25,7 @@ For some of the supported package managers and SCMs additional tools need to be
 * Go (tested with 1.10)
 * Mercurial
 * NPM 5.5.1 and Node.js 8.x (required for this tutorial because we scan `mime-types` which is an NPM project)
+* PHP Composer 1.6.5
```

Comment:
Should we now say `>= 1.5`? Or if we use a fixed version, use 1.6.3 as that's what comes with Ubuntu 18.04 currently?

Reply:
Changed to `>= 1.5`.